pepagent/pepagent.py

```diff
@@ -68,19 +70,25 @@ def upload_project(
         project: peppy.Project,
         namespace: str = None,
         name: str = None,
+        tag: str = None,
         anno: dict = None,
+        update: bool = False,
     ) -> None:
         """
         Upload project to the database
         :param peppy.Project project: Project object that has to be uploaded to the DB
-        :param str namespace: namespace of the project (Default: 'other')
-        :param str name: name of the project (Default: name is taken from the project object)
-        :param dict anno: dict with annotations about current project
+        :param namespace: namespace of the project (Default: 'other')
+        :param name: name of the project (Default: name is taken from the project object)
+        :param tag: tag (or version) of the project
+        :param anno: dict with annotations about current project
+        :param update: boolean value if project hase to be updated
```

Comment:
Do these need type annotations? Just basing off of `peppy` methods: https://github.com/pepkit/peppy/blob/5b519de4f5ca4c5b598f0af7d06cfb7178bb04ee/peppy/project.py#L66

Reply:
@nsheff ? In new Python version it's not required. Do we have to do it?

Reply:
I don't have strong feelings... what did we do in peppy, etc? In general I would suggest including types.